rules/aip0140/prepositions.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 		https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0140
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/googleapis/api-linter/lint"
+	"github.com/jhump/protoreflect/desc"
+	"github.com/scylladb/go-set/strset"
+)
+
+var noPrepositions = &lint.FieldRule{
+	Name: lint.NewRuleName("core", "0140", "prepositions"),
+	URI:  "https://aip.dev/140#prepositions",
+	LintField: func(f *desc.FieldDescriptor) (problems []lint.Problem) {
+		banned := strset.New(
```

Comment:
I am not sure if using `go-set` package would save more time than importing and maintaining the package in Google3. The package is currently not in Google3.

Reply:
Oh, ugh. That makes sense.
Is there an equivalent package that _is_ imported into google3?

Reply:
`bitbucket.org/creachadair/stringset` is a very popular one (>1000 imports) in Google3.

Reply:
Sold, thanks!

Reply:
Done. Thanks.